src/Components/HandleUserChoice.js

```diff
@@ -0,0 +1,54 @@
+
+import React from 'react';
+
+const HandleUserChoice = (props) => {
+  const handleCloseModal = () => {
+
+    let valueSelected="";
+    if (document.getElementById('Today').checked) {
```

Comment:
This is not aligned with the recommended approach in React for handling such operations. Please read more about Ref and let's try to use it instead.